BE/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "epm-mentoring-program-2020-be",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc -b",
+    "start": "./node_modules/.bin/nodemon src/index.ts",
```

Comment:
Im sure that we could use scripts without relative paths in both FE and BE parts like
`nodemon src/index.ts`